src/AppInstallerSharedLib/AppInstallerStrings.cpp

```diff
@@ -867,4 +867,17 @@ namespace AppInstaller::Utility
         THROW_HR_IF(E_UNEXPECTED, !StringFromGUID2(value, buffer, ARRAYSIZE(buffer)));
         return ConvertToUTF8(buffer);
     }
+
+    bool IsDwordFlagSet(const std::string& value)
+    {
+        if (std::empty(value))
+        {
+            return false;
+        }
+
+        DWORD dwordValue = std::stoul(value);

```

Comment:
This throws an exception if value is not all good.  I think it might be better to reuse the method that version parsing uses (see `Version::Part::Part(const std::string& part)`), which is `strtoull`.  Alternatively, catch the exception and make that a false result.